includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -42,6 +42,8 @@ public function sanitize() {
 				continue;
 			}
 
+			// TODO: add a filter to kill the iframe if not https
```

Comment:
Looks like they've moved this here: https://github.com/ampproject/amphtml/issues/2261

Let me see if I can work out an architecture to support both. I can very easily see getting requests for a fallback or just removing it altogether.